docs/tutorial/part-four/index.md

```diff
@@ -293,64 +293,64 @@ Page queries live outside of the component definition -- by convention at the en
 
 [StaticQuery](/docs/static-query/) is a new API introduced in Gatsby v2 that allows non-page components (like our `layout.js` component), to retrieve data via GraphQL queries.
 
```

Comment:
```suggestion
Let's use its newly introduced hook version — [`useStaticQuery`](/docs/use-static-query/)
```

Adding some context so that the jump from `StaticQuery` in the introduction to `useStaticQuery` in the next line is more obvious